### ISSUE TITLE
Translate micmac From C++ to JavaScript

### ISSUE THREAD

**Issue by polyzer** [micmacIGN/micmac#75]
Hello, guys! What about translating micmac to JavaScript? Can we do that?

**Comment by ToroCat** [comment]
It must be a super hard work, besides the performance is a big problem.

**Comment by polyzer** [comment]
May be we can use Emscripten? (https://github.com/kripken/emscripten)
Let's trying!

**Comment by luc-girod** [comment]
No. To Java could have a point (no idea what, but maybe). To Javascript, which is a webdesign language, no. Closing the thread.
